### ISSUE TITLE
[V2 Cold Start] Investigate Tiered Jitting

### ISSUE THREAD

**Issue by alrod** [Azure/azure-functions-host#2515]
Testing ASP.NET Core 2.1.0 preview + Tiered Jitting

**Comment by paulbatum** [comment]
Our first round of testing this had some issues. Not tracking this as important for V2 GA at this point in time.

**Comment by alrod** [comment]
Tiered Jitting is already added

**Comment by paulbatum** [comment]
@alrod Are you sure? I thought this was off by default in .NET Core 2.2 - did we turn it on for functions?

**Comment by alrod** [comment]
@paulbatum you are right, I thought @safihamid enabled it for some reason. Just chatted with him and he measured it awhile back and it was making our cold start worse. Maybe there are optimizations in .net core 3.0 for it. Reopened.

**Comment by kshyju** [comment]
Closing this issue as v2 is no longer supported. Since this was originally reported, there have been significant improvements to JIT performance in Azure Functions.
